sdk/core/System.ServiceModel.Rest/src/PipelineOptions.cs

```diff
@@ -15,6 +15,11 @@ public class PipelineOptions
         /// </summary>
         public CancellationToken CancellationToken { get; set; }
 
+        /// <summary>
+        /// TBD.
+        /// </summary>
+        public ResultErrorOptions ResultErrorOptions { get; set; } = ResultErrorOptions.Default;
```

Comment:
A better enum solution could maybe make this go away.  We need it to move ErrorOptions up to the base type from RequestContext to implement selectively throwing RFE's from protocol methods.